src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -290,6 +290,9 @@ def load_arguments(self, _):
         c.argument('node_resource_group')
         c.argument('enable_defender', action='store_true')
         c.argument('defender_config', validator=validate_defender_config_parameter)
+        c.argument('disable_disk_driver', action='store_true')
+        c.argument('disable_file_driver', action='store_true')
+        c.argument('disable_snapshot_controller', action='store_true')
```

Comment:
Just out of curiosity, may I ask why is the design of this set of parameters not consistent with the following combination to support the `enable_xxx` parameters?

Reply:
These drivers are enabled by default. So when creating, we just need to provide disable flag. In addition, here is the preview PR https://github.com/Azure/azure-cli-extensions/pull/4742. It includes more context. We have discussed before.

Reply:
Got it, thanks~